excel_import_export/wizard/import_xlsx_wizard.py

```diff
@@ -98,8 +102,8 @@ def default_get(self, fields):
             if not template.datas:
                 act = self.env.ref("excel_import_export.action_xlsx_template")
                 raise RedirectWarning(
-                    _('File "%s" not found in template, %s.')
-                    % (template.fname, template.name),
+                    _('File "%(fname)s" not found in template, %(name)s.')
+                    % {"fname": template.fname, "name": template.name},
```

Comment:
```suggestion
                    _('File "%(fname)s" not found in template, %(name)s.', fname=template.fname, name=template.name),
```
For future reference.